include/criterion/internal/deprecation.h

```diff
@@ -32,4 +32,12 @@
 #  define CR_DEPRECATED_(Msg) _Pragma(#Msg)
 # endif
 
+# ifdef __GNUC__
+#   define CR_DEPRECATED_MEMBER(Member) Member __attribute__((deprecated))
+# elif defined (_MSC_VER)
+#   define CR_DEPRECATED_MEMBER(Member) __pragma deprecated (Member)
```

Comment:
Further research shows that doing this particular piece of code will, in fact, deprecate the `Member` identifier regardless of the context. Deprecating the word `failed` would be pretty bad, so I think that we'll just have to remove the MSVC-specific part.